app/src/main/java/org/schabi/newpipe/local/feed/service/FeedLoadService.kt

```diff
@@ -183,28 +183,23 @@ class FeedLoadService : Service() {
 
         subscriptions
             .take(1)
-
```

Comment:
Why did you remove the lines? Or did a plugin do it?

Reply:
This was ktlint going off from new lint additions in newer versions. Doing formatKtlint did this.

Reply:
Not a problem, I think